game.cpp

```diff
@@ -291,7 +312,9 @@ void game::tick()
 {
   if(has_collision(*this))
     {
-      kill_losing_player(*this);
+      //kill_losing_player(*this);
```

Comment:
One can safely delete this line. I feel it is friendly to leave such an obsolete line in, yet superior to remove it. I'll let a newbie delete it :-)